manifests/templates/imageio.yaml.in

```diff
@@ -0,0 +1,53 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: imageio-deployment
+  namespace: {{ .Namespace }}
+  labels:
+    cdi.kubevirt.io/testing: ""
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: imageio
+  template:
+    metadata:
+      labels:
+        app: imageio
+        cdi.kubevirt.io/testing: ""
+    spec:
+      serviceAccountName: cdi-sa
+      containers:
+      - name: imageiotest
+        image: machacekondra/imageiotest
+        imagePullPolicy: {{ .PullPolicy }}
+        ports:
+        - containerPort: 12345
+      - name: fakeovirt
+        image: machacekondra/fakeovirt:v3
```

Comment:
Is this an easy to produce image?  ie. just a Dockerfile?  If so, that should probably be a part of CDI.  If it's complex, then an external repo is okay but you must comment in the manifest where the source code to produce the image can be found.